src/ui/components/DownloadsManagerView/index.tsx

```
@@ -185,93 +208,146 @@ const DownloadsManagerView = () => {
+        {t('downloads.title')}
       </Box>
+
       <Box
         display='flex'
-        justifyContent='space-between'
         alignItems='center'
-        marginBlock={8}
-        padding={24}
+        pi={24}
+        pbe={16}
       >
```

ESLint

[error] Replace `⏎········display='flex'⏎········alignItems='center'⏎········pi={24}⏎········pbe={16}⏎······` with `·display='flex'·alignItems='center'·pi={24}·pbe={16}`

(prettier/prettier)